public/pages/CreateTransform/components/ReviewDefinition/ReviewDefinition.tsx

```diff
@@ -17,7 +17,7 @@ import React, { Component } from "react";
 import { EuiFlexGrid, EuiSpacer, EuiFlexItem, EuiText, EuiAccordion } from "@elastic/eui";
 import { ContentPanel, ContentPanelActions } from "../../../../components/ContentPanel";
 import { ModalConsumer } from "../../../../components/Modal";
-import { TransformGroupItem } from "../../../../../models/interfaces";
+import { TransformGroupItem, DimensionItem } from "../../../../../models/interfaces";
```

Comment:
I think `DimensionItem` is not used anywhere in this class, right? We can clean it up later.